docs/release-notes/releaseposts/2024121101.md

```diff
@@ -0,0 +1,15 @@
+---
+draft: false
+date: 2024-12-11
+---
+
+# Media Connectors - OAuth JWT Bearer authentication
+
+![rn_icon](/assets/icon-GraFx-Studio.svg)
+
+<!-- more -->
+
+## Add support OAuth JWT Bearer authentication flow
```

Comment:
This sentence is not correct. I would say "Support for OAuth ...". Or maybe it doesn't even need a separate header, as the release note title already mentions this.

Reply:
Left out the header, it's indeed double